docs/data/date-pickers/custom-components/Tabs.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+import dayjs, { Dayjs } from 'dayjs';
+import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
+import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
+import { TextField, Box } from '@mui/material';
+import { DateTimePicker } from '@mui/x-date-pickers/DateTimePicker';
+import {
+  DateTimePickerTabs,
+  DateTimePickerTabsProps,
+} from '@mui/x-date-pickers/DateTimePicker/DateTimePickerTabs';
```

Comment:
```suggestion
} from '@mui/x-date-pickers/DateTimePicker';
```
An import of this depth will not work with some building process because we don't provide a `package.json` in the folder.
If we are using this component in a demo, we should export it from `@mui/x-date-pickers/DateTimePicker` and `@mui/x-date-pickers`

Reply:
Done. Re-exported `DateTimePickerTabs` in `DateTimePicker/index`